adobe/spectrum-web-components
# fix(overlay): remove flex display for dialog

## Description

Display flex would affect the positioning of the dialog in Safari. 

BEFORE: 
https://github.com/user-attachments/assets/a284dee0-9102-4944-87e8-6e052a34ac0f


AFTER: 
https://github.com/user-attachments/assets/8476b0c2-bd7a-4957-8107-c8bfcc1ed393


## Related issue(s)
Fixes https://github.com/adobe/spectrum-web-components/issues/4734

## Motivation and context



## How has this been tested?




- Create a parent element that has `transform: translate()` style applied. 
- Add a child element that opens a tooltip/popover. 
- The popover should render correctly.


## Screenshots (if appropriate)

## Types of changes




## Checklist




## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
